docs/guide/create_a_track.md

```diff
@@ -17,9 +17,9 @@ your build settings should look like that: <br>
 <img src="../assets/create_a_track/build_settings.png" height="512">
 
 ## Apply the scene prefab
-1) In "Assets/Prefabs/" you will find a Prefab called "ScenePrefab".
+1) In "Assets/Prefabs/" you will find a Prefab called "ScenePrefab". (if not maybe you can find it in Assets/Scenes)
 2) Drag and drop this prefab into the Hierarchy tab of your new scene, you should see the prefab appear.
-3) Right click on the prefab and select "Unpack Prefab", this will disociate the object in your scene from the Prefab stored in "Assets/Scenes/".
+3) Right click on the prefab and select "Unpack Prefab" (Unpack -> unpack completely), this will disociate the object in your scene from the Prefab stored in "Assets/Scenes/".
```

Comment:
The issue behind unpacking everything is that if you change a given sub-prefab in the scenePrefab (for instance the RaceCanvas prefab). You will loose the advantage of having a prefab there, the changes won't be applied to this scene.
Fully unpacking the scenePrefab may lead to unwanted behaviors.
For this reason, I think "Prefab -> Unpack" is best.